src/imports.jl

```diff
@@ -138,6 +138,17 @@ end
 
 import Hecke:
     _rational_canonical_form_setup,
+    _block_indices_vals,
+    _solve_X_ker,
+    _jordan_odd_adic,
+    _two_adic_normal_forms,
+    _jordan_2_adic,
+    _normalize,
+    _val,
+    _min_val,
+    hensel_qf,
+    TorQuadMod,
+    gram_matrix_quadratic,
```

Comment:
I don't want to be too annoying, but this should be ordered alphabetically so that we finally know where the imports actually are.